sherpa/astro/io/__init__.py

```diff
@@ -74,26 +92,24 @@
     if ogip_emin <= 0.0:
         raise ValueError(emsg)
 
-if io_opt.startswith('pycrates') or io_opt.startswith('crates'):
-    io_opt = 'crates_backend'
-
-elif io_opt.startswith('pyfits'):
-    io_opt = 'pyfits_backend'
+backend = None
+'''Currently active backend module for astronomy specific I/O.'''
 
-try:
-    importlib.import_module('.' + io_opt, package='sherpa.astro.io')
-    backend = sys.modules['sherpa.astro.io.' + io_opt]
-except ImportError:
-    raise ImportError("""Cannot import selected FITS I/O backend {}.
-    If you are using CIAO, this is most likely an error and you should contact the CIAO helpdesk.
-    If you are using Standalone Sherpa, please install astropy."""
-                      .format(io_opt))
+if backend is None:
```

Comment:
Is this check needed? You've just set `backend` to `None` which implies this is always going to fire, doesn't it?